### PR TITLE
Update plugin names and deprecations for scheduler config

### PR DESCRIPTION
**Pull request by alculquicondor**
Ref kubernetes/kubernetes#95446

Part of kubernetes/enhancements#785

http://git.k8s.io/enhancements/keps/sig-scheduling/785-scheduler-component-config-api

/sig scheduling

### CODE REVIEW

**Comment by alculquicondor** [comment]
/assign @Huang-Wei 
cc @adtac

**Comment by PI-Victor** [comment]
could you please link the kep for this, i'm having trouble understanding where this is?

**Comment by alculquicondor** [comment]
Updated

**Comment by PI-Victor** [comment]
need to clarify some things

/hold

File: content/en/docs/reference/scheduling/config.md
Comment (sftim) [inline content/en/docs/reference/scheduling/config.md:206]: This annotation should also be listed in https://kubernetes.io/docs/reference/labels-annotations-taints/

File: content/en/docs/reference/scheduling/config.md
Comment (sftim) [inline content/en/docs/reference/scheduling/config.md:181]: Optional extra tweak
```suggestion
- `CinderLimits`: Checks that [OpenStack Cinder](https://docs.openstack.org/cinder/) volume limits can be satisfied
```

**Review by sftim — DISMISSED** [review]

File: content/en/docs/reference/scheduling/config.md
Comment (Huang-Wei) [inline content/en/docs/reference/scheduling/config.md:29]: Replace v1beta1 with v1beta2 in L23?

BTW: @sftim as it's now v1beta2, with slightly different API fields, should we also update L7 to v1.22?

File: content/en/docs/reference/scheduling/config.md
Comment (Huang-Wei) [inline content/en/docs/reference/scheduling/config.md:62]:
```suggestion
   feasible nodes were found for the pod. If any postFilter plugin marks a Pod
```

File: content/en/docs/reference/scheduling/config.md
Comment (Huang-Wei) [inline content/en/docs/reference/scheduling/config.md:63]: Prefer not to emphasize/backquote "Schedulable" as it's just an internal state:

```suggestion
   schedulable, the remaining plugins are not called.
```

File: content/en/docs/reference/scheduling/config.md
Comment (Huang-Wei) [inline content/en/docs/reference/scheduling/config.md:208]: (not related with this line)

Update L261 to link to v1beta2.

File: content/en/docs/reference/scheduling/config.md
Comment (alculquicondor) [reply content/en/docs/reference/scheduling/config.md:29]: Better listed the options.

I don't think we should update the min beta version.

File: content/en/docs/reference/scheduling/config.md
Comment (alculquicondor) [reply content/en/docs/reference/scheduling/config.md:62]: the Pod.

File: content/en/docs/reference/scheduling/config.md
Comment (alculquicondor) [reply content/en/docs/reference/scheduling/config.md:63]: Done

File: content/en/docs/reference/scheduling/config.md
Comment (alculquicondor) [reply content/en/docs/reference/scheduling/config.md:181]: Done

File: content/en/docs/reference/scheduling/config.md
Comment (alculquicondor) [reply content/en/docs/reference/scheduling/config.md:206]: Done

File: content/en/docs/reference/scheduling/config.md
Comment (alculquicondor) [reply content/en/docs/reference/scheduling/config.md:208]: Done

File: content/en/docs/reference/scheduling/config.md
Comment (sftim) [reply content/en/docs/reference/scheduling/config.md]: Maybe put _schedulable_ in italics to reflect that it is a specific term with meaning to the code.

**Comment by PI-Victor** [comment]
this enhancement is still labelled 'At Risk', no worries tho, once this clears, i will merge myself

/cancel approve

**Comment by PI-Victor** [comment]
ugh...

/approve cancel

**Comment by alculquicondor** [comment]
Note that the exception request was approved: https://groups.google.com/g/kubernetes-sig-release/c/2Zv7SZf-urE/m/jbAQ_zo8AAAJ

File: content/en/docs/reference/labels-annotations-taints.md
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints.md:271]: I prefer to not make “deprecated” part of the fragment identifier.
```suggestion
## scheduler.alpha.kubernetes.io/preferAvoidPods (deprecated) {#scheduleralphakubernetesio-preferavoidpods}
```

File: content/en/docs/reference/scheduling/config.md
Comment (sftim) [inline content/en/docs/reference/scheduling/config.md:63]: Is it PostFilter or postFilter?

File: content/en/docs/reference/labels-annotations-taints.md
Comment (alculquicondor) [reply content/en/docs/reference/labels-annotations-taints.md]: Done

File: content/en/docs/reference/scheduling/config.md
Comment (alculquicondor) [reply content/en/docs/reference/scheduling/config.md]: `postFilter` in the context of the configuration file. `PostFilter` in the context of the code interface that plugins implement. So, for the purpose of this doc, it should be `postFilter`. Fixed all the other references.

File: content/en/docs/reference/scheduling/config.md
Comment (alculquicondor) [reply content/en/docs/reference/scheduling/config.md]: Done

**Comment by alculquicondor** [comment]
/hold cancel
Given that the exception was approved

**Pull request merged**
